challenge/src/components/SomeInformationForTimeTravelers.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export default class SomeInformationForTimeTravelers extends React.Component {
+    render() {
+        var date = new Date();
+        var year = date.getFullYear();
+        var month = date.toLocaleString('default', {month: 'long'});
+
+        return (
+            <h1 style={{textAlign: "center"}}>It is {month} {year}</h1>
+        )
+    }
+}
+
+
+
+
+
+// Leaving this here in memory of my poor if statements that got attacked
```

Comment:
its good to remember growth, but if you don't remove this, it will haunt you forever

Reply:
yes i removed it :(